### PR TITLE
Renamed attributes by means of DICOM Dictionary 

### PR DESCRIPTION
**Pull request by che85**
related to #146 

TODO: webapp needs to be adapted to those changes

### CODE REVIEW

File: doc/pm-example-float.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "CodeMeaning": "Mono-exponential diffusion model"
   },
   "SourceImageDiffusionBValues": ["0","1400"],
-  "AnatomicRegionCode": {
+  "AnatomicRegionSequence": {
```
Comment (fedorov) [inline doc/pm-example-float.json]: I think this one should be `AnatomicRegionCodeSequence`, if we want to be consistent.

File: include/dcmqi/SegmentAttributes.h [hunk 2]
```diff
@@ -60,11 +61,11 @@ namespace dcmqi {
     string segmentAlgorithmType;
     string segmentAlgorithmName;
     unsigned recommendedDisplayRGBValue[3];
-    CodeSequenceMacro* anatomicRegion;
-    CodeSequenceMacro* anatomicRegionModifier;
-    CodeSequenceMacro* segmentedPropertyCategoryCode;
-    CodeSequenceMacro* segmentedPropertyType;
-    CodeSequenceMacro* segmentedPropertyTypeModifier;
+    CodeSequenceMacro* anatomicRegionSequence;
+    CodeSequenceMacro* anatomicRegionModifierSequence;
```
Comment (fedorov) [inline include/dcmqi/SegmentAttributes.h]: `*CodeSequence`

File: doc/pm-example-float.json (hunk 1 above)
Comment (fedorov) [inline doc/pm-example-float.json]: Oh, nevermind - you are right! I checked, and it is correct! Interesting ...

File: include/dcmqi/SegmentAttributes.h (hunk 2 above)
Comment (fedorov) [inline include/dcmqi/SegmentAttributes.h]: never mind!

**Comment by che85** [comment]
Yeah I am also not sure, why it's that inconsistent in DICOM itself ....

**Pull request merged**